src/core/consts.c

```diff
@@ -91,7 +100,8 @@ ROX_INTERNAL size_t rox_env_get_analytics_path(char *buffer, size_t buffer_size)
             buffer, buffer_size,
             "http://127.0.0.1:8787",
             "https://qaanalytic.rollout.io",
-            "https://analytic.rollout.io");
+            "https://analytic.rollout.io",
+            "https://fm-analytics.cloudbees.io/impression");
```

Comment:
did you check impression?
i think the code adds the `/impression` part
(so it should probably be "https://fm-analytics.cloudbees.io" like the other urls)